src/drawers/arcDrawer.ts

```diff
@@ -6,6 +6,7 @@ export module Drawers {
 
     constructor(dataset: Dataset) {
       super(dataset);
+      this._className = "arc";
```

Comment:
The types of "rect"s that are drawn in a `Drawers.Rect` are... "rect"s?  It is really redundant I think in this case, whereas we have many different types of "path"s with no good way of differentiation